backend/lib/spree/backend/tabs/product/base.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Spree
+  module Backend
+    module Tabs
+      class Product
+        class Base
+          attr_reader :view_context, :current
+
+          def initialize(view_context:, current: nil)
+            @view_context = view_context
+            @current = current
+          end
+
+          def name
+            raise NotImplementedError
+          end
```

Comment:
We do not use the name anywhere. Do we need it then?

Reply:
We use it into the `active?` method actually.

Reply:
Oh, my bad :)